harvard-cite-them-right-11th-edition.csl

```diff
@@ -124,6 +134,28 @@
       </if>
     </choose>
   </macro>
+
+  <!-- NEW: AI helpers -->
+  <macro name="ai-version">
+    <choose>
+      <if type="software" variable="version" match="all">
+        <text variable="version" prefix=" (" suffix=")"/>
+      </if>
+    </choose>
+  </macro>
+
+  <macro name="ai-prompt">
+    <choose>
+      <if variable="original-title">
+        <group>
+          <text value="prompt: &quot;"/>
+          <text variable="original-title"/>
```

Comment:
I don't think we can accept this hack/workaround to use `original-title` to hold the prompt text. That's not very intuitive.